source/reference/limitations.txt

```diff
@@ -101,9 +101,35 @@ Multiple Clusters
 - Syncing one source cluster to many destination clusters is not
   supported.
 
+System Collections
+------------------
+
+{+c2c-product-name+} does not replicate :ref:`system collections 
+<metadata-system-collections>` to the destination cluster.
+
+If you issue a :dbcommand:`dropDatabase` command on the source cluster,
+this change is not directly applied on the destination cluster. Instead,
+{+c2c-product-name+} drops user collections and views in the destination 
```

Comment:
[nit]

I'd suggest updating this to be "on the destination database..." to be more consistent with the remaining copy.

Reply:
Fixed